### ISSUE TITLE
Please upload this to Kaggle

### ISSUE THREAD

**Issue by vikalpjain91** [namtpham/casia1groundtruth#1]
Hi Namtpham Can you please upload your dataset to Kaggle? I can also do it from your Google Drive but it will be quite unfair. Please upload it there because someone has uploaded the wrong dataset and it is quite confusing.

**Comment by namtpham** [comment]
Hi vikalpjain91. Thank you for your suggestion. As I stated in the description, I am not the owner of the dataset. However, I can upload my generated groundtruth and link to the dataset.

**Comment by vikalpjain91** [comment]
Yes just mention everything because I have spent like 15 days on wrong data set realizing that it was not correct, as original data is nowhere available as far as I have searched so you are the one who got this great responsibility to upload it on Kaggle for the sake of research. In case if you feel you are not the owner you can mention the origin in description and usage license.

**Comment by namtpham** [comment]
Thank you. I also spent lots of time working on this dataset. I will upload it to Kaggle soon!
